doc/manpages/qvm-features.rst

```diff
@@ -341,6 +341,12 @@ Boot the HVM qube via UEFI boot, instead of legacy one. Support for this boot
 mode is experimental and may not work in all the cases. It is ignored for
 non-HVM qubes.
 
+Custom and user specific features
+---------------------------------
+
+Features with `x-` prefix is dedicated to end users and is guarantied to never
+be used for Qubes OS future internal features.
```

Comment:
Features with the `x-` prefix are specifically meant for end users and will never
be used for internal Qubes OS features.

---

Grammar and some rewording to make it easier to understand.

Maybe it should not be `user specific features` but `3rd party` as mentioned in https://github.com/QubesOS/qubes-issues/issues/9557#issuecomment-2479008750